website/docs/usage/large-language-models.mdx

```diff
@@ -357,6 +357,7 @@ evaluate the component.
 
 | Component                                                               | Description                                                                                                       |
 | ----------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| [`spacy.EntityLinker.v1`](/api/large-language-models#summarization-v1)  | The entity linking task prompts the model to link all entities in a given text to entries in a knowledge base.    |
```

Comment:
TODO: fix API link

Reply:
Fixed in 99f23e87bd3878b303d4a8b62023bcd7899950b3.